Packs/TAXIIServer/Integrations/TAXIIServer/TAXIIServer_description.md

```diff
@@ -14,16 +14,26 @@ The collections are defined by a JSON object in the following format:
 
 ## How to Access the TAXII Service
 
-To view the available TAXII services, visit the discovery service in one of the following options:
- 
-- **https://*demisto_address*/instance/execute/*instance_name/taxii-discovery-service*** 
-- **http://*demisto_address*:*listen_port/taxii-discovery-service***
+(For Cortex XSOAR 6.x) Use one of the following options:
+
+- **https://*demisto_address*/instance/execute/*instance_name/taxii-discovery-service**
+- **http://*demisto_address*:*listen_port/taxii-discovery-service**
+
+(For Cortex XSOAR 8 and Cortex XSIAM) Use one of the following options:
+- **https://ext-<cortex-xsiam-address>/xsoar/instance/execute/<instance-name>/{taxii2_api_endpoint}/**
+- **http://demisto_address:listen_port/{taxii2_api_endpoint}/**
```

Comment:
(For Cortex XSOAR 8) Use one of the following options:
   - `https://ext-<XSOAR-tenant>.crtx.<region>.paloaltonetworks.com/xsoar/instance/execute/<instance-name>`
   - **http://xsoar_address:listen_port/{taxii2_api_endpoint}/**
   (For Cortex XSIAM) Use one of the following options:
   - **https://ext-<XSIAM-tenant>.crtx.<region>.paloaltonetworks.com/xsiam/instance/execute/<instance-name>**
   -  **http://xsiam_address:listen_port/{taxii2_api_endpoint}/**